src/PlugWallet/index.ts

```diff
@@ -111,14 +84,9 @@ class PlugWallet {
     this.walletId = walletId;
     this.orderNumber = orderNumber;
     this.walletNumber = walletNumber;
-    this.assets = assets;
     this.icnsData = icnsData;
     this.identity = identity;
-    this.accountId = getAccountId(identity.getPrincipal());;
```

Comment:
is this `accountId` accesible from elsewhere?

Reply:
the behavior is the same as I exposed a getter for it which is called exactly the same way. 
And in no other place it's updated so it should be OK!